coffeecard/CoffeeCard.WebApi/Models/Statistic.cs

```diff
@@ -25,16 +25,13 @@ public class Statistic
 
         public static DateTime GetSemesterStart(DateTime currentTime)
```

Comment:
This is not on you but these methods do not make sense to have on a model class; `Statistics`. It would be much more meaning full to move these utility methods to a separate static helper class, 

Also to make it more clear for future users, we should probably add some documentation. I've added a suggestion

```suggestion
        /// <summary>
        /// Get the start of date of the semester.
        /// When <paramref name="currentTime"/> is in July (Month 7) or greater, the semester start is July 1st or when earlier than July, the semester start is the date of the last Monday of January.
        /// </summary>
        public static DateTime GetSemesterStart(DateTime currentTime)
```

Reply:
> This is not on you but these methods do not make sense to have in a model class; `Statistics`. It would be much more meaning full to move these utility methods to a separate static helper class,

I agree. This should be added as an issue.